Reneiski_Homework1/ReneiskiSelenium11/Utilities/Utilities.cs

```diff
@@ -0,0 +1,56 @@
+﻿using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+
+namespace ReneiskiSelenium11.Utilities
+{
+    public class Utility
```

Comment:
what is a purpose of having this class? if it's a kind of extension of IWebElement - please create a wrapper